overlays/ghc-packages.nix

```diff
@@ -69,8 +69,9 @@ let
       libiserv     = "libraries/libiserv";
       template-haskell = "libraries/template-haskell";
       iserv        = "utils/iserv";
-      iserv-proxy  = "utils/iserv-proxy";
+    } // final.lib.optionalAttrs (builtins.compareVersions ghcVersion "9.4" < 0) {
       Win32        = "libraries/Win32";
+      iserv-proxy  = "utils/iserv-proxy";
```

Comment:
This needs a comment.Also, what happened to Win32?

Reply:
Comments added